Doc/library/asyncio-stream.rst

```diff
@@ -126,10 +126,12 @@ Stream functions
 StreamReader
 ============
 
-.. class:: StreamReader(limit=None, loop=None)
+.. class:: StreamReader(limit=_DEFAULT_LIMIT, loop=None)
 
    This class is :ref:`not thread safe <asyncio-multithreading>`.
 
+   The *limit* argument default value is 2 ** 16 (64 KiB)
```

Comment:
You can mention it like this so that it is more clear:

`The *limit* argument's default value is set to _DEFAULT_LIMIT which is 2**16 (64 KiB)`.

Reply:
Done, thx :)